src/beneuro_data/cli.py

```diff
@@ -678,6 +742,119 @@ def upload_session(
     return True
 
 
+@app.command()
+def up(
+    session_name: Annotated[
+        str, typer.Argument(help="Name of session: M123_2000_02_03_14_15")
+    ],
+    include_behavior: Annotated[
+        bool,
+        typer.Option(
+            "--include-behavior/--ignore-behavior",
+            "-b/-B",
+            help="Download behavioral data (-b) or not (-B)."
+        ),
+    ] = True,
+    include_ephys: Annotated[
+        bool,
+        typer.Option(
+            "--include-ephys/--ignore-ephys",
+            "-e/-E",
+            help="Download ephys data (-e) or not (-E)."
+        ),
+    ] = False,
+    include_videos: Annotated[
+        bool,
+        typer.Option(
+            "--include-videos/--ignore-videos",
+            "-v/-V",
+            help="Download video data (-v) or not (-V)."
+        ),
+    ] = False,
+    include_extra_files: Annotated[
+        bool,
+        typer.Option(
+            "--include-extra-files/--ignore-extra-files",
+            help="Upload extra files that are created by the experimenter or other software.",
+        ),
+    ] = True,
+    rename_videos_first: Annotated[
+        bool,
+        typer.Option(
+            "--rename/--no-rename",
+            help="Rename videos before validating and uploading.",
+        ),
+    ] = True,
+    rename_extra_files_first: Annotated[
+        bool,
+        typer.Option(
+            "--rename-extra/--no-rename-extra",
+            help="Rename extra files (e.g. comment.txt) before validating and uploading.",
+        ),
+    ] = True,
+    processing_level: Annotated[
+        str, typer.Argument(help="Processing level of the session. raw or processed.")
+    ] = "raw",
+):
+    """
+    Upload (raw) experimental data to the remote server.
+
+    Example usage to upload everything of a given session:
+        `bnd up M017_2024_03_12_18_45 -ev`
```

Comment:
It might be a bit confusing that behavior is uploaded by default, but not ephys and videos, and `-ev` actually means `-bev`. It definitely confused me now.
What do you think about changing the default to false for behavior as well, then using `-bev`?

Reply:
disagreed! behaviour needs to be default, because we do it everyday for every animal: no one like to type more than they have to. Video and ephys  however are occasional.